tmp/.gitignore

```diff
@@ -1,4 +1,10 @@
 # Ignore everything in this directory
-*
+/*
```

Comment:
is the beginning slash the root directory from this app?  it's been a while ...

Reply:
Good eye, yes, the docs I was reading used it. 

https://git-scm.com/docs/gitignore

If you look at the last example?